examples/nextjs-spa/src/pages/verification.tsx

```diff
@@ -15,83 +15,49 @@ import { AxiosError } from "axios"
 // We will use UserAuthCard from Ory Elements to display the verification form.
 import Layout from "@/components/layout"
 import { UserAuthCard } from "@ory/elements"
-import { QueryParams } from "../pkg/helpers"
-import { HandleError } from "../pkg/hooks"
+import { SetUriFlow } from "@/pkg/helpers"
+import { HandleError } from "@/pkg/hooks"
 import { NextPageWithLayout } from "./_app"
 
 const Verification: NextPageWithLayout = () => {
   const [flow, setFlow] = useState<VerificationFlow | null>(null)
 
-  const handleError = HandleError()
-
   // Get ?flow=... from the URL
   const router = useRouter()
 
   const flowId = String(router.query.flow || "")
   const returnTo = String(router.query.return_to || "")
 
-  const getVerificationFlow = useCallback(
+  const getFlow = useCallback(
     (id: string) =>
       ory
         .getVerificationFlow({ id })
-        .then(({ data }) => {
-          setFlow(data)
-        })
-        .catch((err: AxiosError) => handleError(err))
-        .catch((err: AxiosError) => {
-          switch (err.response?.status) {
-            case 410:
-              // Status code 410 means the request has expired - so let's load a fresh flow!
-              return router.push("/verification")
-            case 403:
-              // Status code 403 implies some other issue (e.g. CSRF) - let's reload!
-              return router.push("/verification")
-          }
-          if (err.response?.status === 401) {
-            router.push("/login")
-          } else {
-            router.push({
-              pathname: "/error",
-              query: {
-                error: JSON.stringify(err, null, 2),
-                id: err.response?.data.error?.id,
-                flowType: router.pathname,
-              },
-            })
-          }
-        }),
-    [router, handleError],
+        .then(({ data }) => setFlow(data))
+        .catch(handleError),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I'm not sure if we should disable linter rules like this.
Since the functions are memoized through the `useCallback` function, it should be alright to add them to this dependency list.